README.md

```diff
@@ -123,6 +117,18 @@ This section describes guidelines that _must_ be followed when applying changes
                 sh:path <your_prefix>:general ],
     ```
 
+## CI pipeline
+
+The CI/CD pipeline is defined in the `.github/workflows` directory. The pipeline is triggered on every push to the repository as defined in the workflow. 
+ 
+### Generate VARIABLES.md (overview of used properties in the SHACL files)
+The file VARIABLES.md will be generated automatically when a push to a non-main branch is executed. This file is existent in every subdirectory once there is a SHACL file containing properties. This should help to get a fast overview of the properties used in the SHACL files.
+
+>NOTE: the VARIABLES.md file should not be changed since it will be overwritten automatically. 
+
+### Check syntax of Turtle files
+The pipeline checks the syntax of the Turtle files (`*.ttl`) by loading a RDF graph. If the Turtle file is not correct the pipeline fails with a detailed error message.
+
```

Comment:
Should we add notes to execute the check locally using python venv?
```
$ python3 -m venv .venv
$ source .venv/bin/activate
(.venv) $ python3 -m pip install --upgrade pip
(.venv) $ python3 -m pip install -r requirements_develop.txt
# execute check from CI
```

Reply:
added this into the readme